app/src/main/java/com/teamagam/gimelgimel/app/model/ViewsModels/messages/MessageDetailViewModel.java

```diff
@@ -35,6 +36,10 @@ private boolean isSelectedMessageOfType(@Message.MessageType String messageType)
         return mSelectedMessageModel.getSelected().getType().equals(messageType);
     }
 
+    public void drawMessageOnMap(MessagesDetailBaseGeoFragment.GeoMessageInterface drawMessageOnMapInterface) {
```

Comment:
organize method arrangement. public method should appear before protected/private !fix